src/game.cpp

```diff
@@ -9732,12 +9732,12 @@ void game::on_move_effects()
                 }
             }
         }
-        if( u.lifetime_stats.squares_walked % 160 == 0 ) {
+        if( u.lifetime_stats.squares_walked % 160 == 0 ) { //  passive power gen
             if( u.has_bionic( bionic_id( "bio_torsionratchet" ) ) ) {
                 u.charge_power( 1 );
             }
             for( const bionic_id &bid : u.get_bionic_fueled_with( muscle ) ) {
```

Comment:
I was looking at this PR and was confused by this second loop. Is this second loop to support legacy code or something? 

It looks like it would just double-count the passive power gen -- but I can't find any bionics in json (mainline, at least) that seems to provide a 'fuel_options' to be returned by `Character::get_bionic_fueled_with`. 

Am I missing something? 

Also -- is there a better place to ask questions like this?